TangoRosStreamer/tango_ros_node/src/main/java/eu/intermodalics/tango_ros_node/TangoRosNode.java

```diff
@@ -64,6 +64,8 @@ public TangoRosNode(String libName) {
     @Override
     public native int shutdown();
 
+    public native boolean saveMap(String mapName);
```

Comment:
I have a doubt about this.
I think it's OK to have a separate method in the node for this purpose, but I'm not sure if this is the "ROS way" of achiving this (e.g. exposing a service that can be called is another option). 
Perhaps exposing a service would make the code more complex than just having a method like this, and I'm not sure if that would have clear benefits in the future. Maybe @adamantivm has some opinion about that.